.github/.wordlist.txt

```diff
@@ -977,6 +979,7 @@ percentageLiftValue
 periodicQueryTimeout
 pexpect
 pickString
+picocom
```

Comment:
picocom is not a word. Use backticks in md files for things that are commands or code.

Reply:
Thanks for pointing it out. I fixed it